frontend/src/types/org.ts

```diff
@@ -39,7 +39,9 @@ export type OrgData = {
   bytesStoredUploads: number;
   bytesStoredProfiles: number;
   usage: { [key: YearMonth]: number } | null;
+  /* Actual total time used, including time to stop the crawl */
```

Comment:
I think crawlExecSeconds also includes any gifted/extra time used - it's basically just a total of all crawl execution time for the month regardless of what quota bucket it would go in (or not, as for overage when stopping crawls)